packages/itwin/quantity-formatting/src/components/quantityformat/FormatSelector.tsx

```diff
@@ -31,53 +29,92 @@ export const FormatSelector: React.FC<FormatSelectorProps> = ({
   onListItemChange,
 }) => {
   const { translate } = useTranslation();
+  const [searchTerm, setSearchTerm] = React.useState("");
   const formatListId = React.useId();
```

Comment:
do we really need these ids? i havent seen enough react 18 usage of this across our ecosystem to justify it